demes/demes.py

```diff
@@ -163,8 +163,8 @@ class Epoch:
         If ``start_size != end_size``, the population size changes
         monotonically between the start and end times.
     :ivar str size_function: The size change function. This is either
-        ``constant`` or ``exponential``, though it is possible that
-        additional values will be added in the future.
+        ``constant``, ``exponential`` or ``linear``, though it is possible
+        that additional values will be added in the future.
```

Comment:
It would be good for the docs to state that if start_size != end_size then the default behavior is exponential, now that there are multiple options for changing population sizes.

Reply:
I can change an old warning that is below to the note that when  `start_size != end_size` then it is exponential size function.